docs/blog/2019-07-01-extending-components/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Extending Shadowed Components
+date: 2019-07-01
+author: John Otander
+excerpt: "Extending components when using Component Shadowing can be a powerful pattern for making small changes."
+tags:
+  - themes
+---
+
+# Extending Components in Themes
```

Comment:
This will create a duplicate h1 since the title will already be rendered. Can we remove it?